components/avatar/avatar.html

```diff
@@ -76,29 +76,68 @@
+              style="width: 1020px; height: 250px; border:0; transform: scale(1); overflow:hidden;"
+              sandbox="allow-scripts allow-same-origin">
+            </iframe>
+          </code>
+        </div>
+        <div class="sub-heading">
+          <strong>Square Avatar</strong>
+        </div>
+        <div class="sub-head-content">
+          Image avatar can be created by using <code>img</code>, <code>img</code>, <code>src</code>, <code>srcSet</code>
+          to the component.
+        </div>
+        <div class="practical avatar-box">
+          <img src="/images/img-avatar-1.jpeg" alt="image 1" class="avatar square-avatar">
+          <img src="/images/img-avatar-2.jpeg" alt="image 1" class="avatar square-avatar">
+          <img src="/images/img-avatar-3.jpeg" alt="image 1" class="avatar square-avatar">
+        </div>
+        <div class="code-box">
+          <code class="html">
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=a11y-dark&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Fira+Code&fs=14px&lh=152%25&si=false&es=2x&wm=false&code=%253Cimg%2520src%253D%2522%252Fimages%252Fimg-avatar-1.jpeg%2522%2520alt%253D%2522image%25201%2522%2520class%253D%2522avatar%2520square-avatar%2522%253E%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%253Cimg%2520src%253D%2522%252Fimages%252Fimg-avatar-2.jpeg%2522%2520alt%253D%2522image%25201%2522%2520class%253D%2522avatar%2520square-avatar%2522%253E%250A%253Cimg%2520src%253D%2522%252Fimages%252Fimg-avatar-3.jpeg%2522%2520alt%253D%2522image%25201%2522%2520class%253D%2522avatar%2520square-avatar%2522%253E"
+              style="width: 1020px; height: 250px; border:0; transform: scale(1); overflow:hidden;"
+              sandbox="allow-scripts allow-same-origin">
+            </iframe>
+          </code>
+        </div>
+        <div class="sub-heading">
+          <strong>Size Avatar</strong>
+        </div>
+        <div class="sub-head-content">
+
+        </div>
+        <div class="practical avatar-box">
+          <img src="/images/img-avatar-1.jpeg" alt="image 1" class="avatar size-1">
+          <img src="/images/img-avatar-2.jpeg" alt="image 1" class="avatar size-2">
+          <img src="/images/img-avatar-3.jpeg" alt="image 1" class="avatar size-3">
```

Comment:
What I like to do for alt text is describe what the image is about rather than putting "image 1". Screen readers will read exactly what we put in alt text. In my case, I have used "avatar image" as alt text.